converter/shieldhit/beam.py

```diff
@@ -12,6 +12,40 @@ class BeamSourceType(Enum):
     FILE = "file"
 
 
+@unique
+class StraggleModel(Enum):
```

Comment:
```suggestion
class StragglingModel(Enum):
```

Only Straggling has some meaning for physicists